### DISCUSSION TITLE
Отладка кода.

### DISCUSSION THREAD

**Discussion by Kukushechka7-ezx** [Sokol-Games-GmbH/PixelCombats.PublicApi#103] [Ideas]
Сделать отладку кода, чтобы прямо выводило ошибки если они есть

**Comment by kkohno** [comment]
что значит выводило ошибки? есть реалтайм ошибки и ошибки компиляции... а вот язык JavaScript не строго типизирован и компилится, как писали по этому поводу в луркморе, даже если писать код прыгая задницей по клавиатуре - жопоязык))))

Опишите подробнее какой инструмент хотите видеть

**Reply by forwork10** [reply]
он таких слов не знает, вы что

**Reply by kkohno** [reply]
ну гугл в помошь, узнает новое, сможет лучше делать режимы

**Reply by Kukushechka7-ezx** [reply]
"ну гугл в помощь", я видел там что можно пофиксить мерцание блоков, наложение текстур

**Comment by Kukushechka7-ezx** [comment]
Ошибки самого апи, или предупреждения, типо вот тут не найдено команды и создана новая

**Reply by kkohno** [reply]
напишите подробнее, с кодом, не совсем понятно... если имеется ввиду ошибка времени компиляции - то такое джаваскрипт не потдерживает... максимум, когда будет апи 2.0 то там возможно будет файл выгрузки модуля апи, для потдержки intelisence

**Reply by Kukushechka7-ezx** [reply]
Позже ок

**Comment by Kukushechka7-ezx** [comment]
Посже сделаем, ок
